### PR TITLE
Upload the EHR file upload documentation

### PR DESCRIPTION
**Pull request by simardeep1792**
Uploading the documentation. Requesting Review

### CODE REVIEW

**Comment by simardeep1792** [comment]
Requesting @lucbelliveau review on the changes for the docker image.

File: paradire/patient_browser_patch/default.json5 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     server: {
         type: 'Unknown',
-        url: 'http://34.95.11.65:8080/fhir',
+        url: 'http://localhost:8080/fhir',
```
Comment (lucbelliveau) [inline paradire/patient_browser_patch/default.json5:4]: My bad for leaving my IP in there!  Thanks for fixing it!  :)

File: paradire/setup_patient_browser.sh [hunk 2]
```diff
@@ -1,52 +1,57 @@
...
-    git clone https://github.com/smart-on-fhir/patient-browser.git
-    ## Install dependencies and build project
-    (cd patient-browser && NODE_ENV=production npm install colors request commander@2.15.1 && npm ci && NODE_ENV=production npm run build)
-fi
+# Navigate to the "patient-browser" directory
+cd "$HOME/patient-browser"
 
-cd patient-browser
+# Create a Dockerfile
+cat <<EOL > Dockerfile
```
Comment (lucbelliveau) [inline paradire/setup_patient_browser.sh:23]: Didn't we say we'd move this to a actual `Dockerfile` instead, in the patient_browser_patch dir?

File: paradire/setup_patient_browser.sh [hunk 3]
```diff
@@ -1,52 +1,57 @@
...
+# Install dependencies and build project
+RUN npm install colors request commander@2.15.1
+RUN npm ci
+RUN NODE_ENV=production npm run build
+
+# Configure the browser
+RUN mv default.json5 ./dist/config/
+
+# Disable the broken FHIR viewer
+RUN sed -i 's/enabled: true/enabled: false/g' ./dist/config/default.json5
```
Comment (lucbelliveau) [inline paradire/setup_patient_browser.sh:44]: This is not required as this change is already done in the committed version in the patch dir.

**Pull request merged**